// Pending implementation: global error boundaries and retry/backoff UX
describe.skip('API error boundaries', () => {
  test.todo('network error surfaces with retry button');
  test.todo('retry triggers refetch with backoff');
});


